reflectapi-derive/src/tokenizable_schema.rs

```diff
@@ -238,6 +240,35 @@ impl<'a> ToTokens for TokenizableEnum<'a> {
                 parameters: vec![#(#parameters),*],
                 representation: #representation,
                 variants: vec![#(#variants),*],
+                codegen_config: #codegen_config,
+            }
+        });
+    }
+}
+
+pub(crate) struct TokenizableLanguageSpecificTypeCodegenConfig<'a>(
```

Comment:
A proc macro noob at the moment. But having to manually transforming everything to tokens manually is fairly brutal? I'm hoping there is an easier way